src/schema/v2/me/deleteMultipleUserInterestsMutation.ts

```diff
@@ -0,0 +1,62 @@
+import { GraphQLList, GraphQLString } from "graphql"
```

Comment:
I think the 'multiple' in the name might be redundant, if the name already is plural, `deleteUserInterestsMutation`